server/routes/universities.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const router = express.Router();
+const { BadRequest, GeneralError } = require('../utils/errors');
+const Course = require("../models/courses");
+const University = require('../models/universities');
+
+//GET all universities with populated courses
+router.get("/", async function(req, res, next) {
+    const allUniversities = await University.find()
```

Comment:
Did you forget to add a .populate ?

Reply:
Yeah it looks like I did. Before I thought I was seeing if I excluded the `{}` inside of the find function it was returning a populated list of courses. But I just tested it again and that's not the case!